### PR TITLE
Hardware and Virtual Machine tools

### PR DESCRIPTION
**Pull request by keachi**
install hardware and virtual machine management tools

### CODE REVIEW

File: hw_vm_tools/tasks/installation.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    state: absent
+  with_items: '{{ hw_vm_tools_packages_remove }}'
+  when: hw_vm_tools_packages_remove is defined
+
+- name: remove unused tools on virtual machines
+  package:
+    name: '{{ item }}'
+    state: absent
+  with_items: '{{ hw_vm_tools_vm_packages_remove }}'
+  when: hw_vm_tools_vm_packages_remove is defined
```
Comment (karras) [inline hw_vm_tools/tasks/installation.yml]: Maybe add a check for the virtualization_role here as well?

**Review by karras — CHANGES_REQUESTED** [review]

File: hw_vm_tools/tasks/installation.yml (hunk 1 above)
Comment (keachi) [reply hw_vm_tools/tasks/installation.yml]: Makes sense.

File: hw_vm_tools/tasks/installation.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    name: '{{ item }}'
+    state: absent
+  with_items: '{{ hw_vm_tools_packages_remove }}'
+  when: hw_vm_tools_packages_remove is defined
+
+- name: remove unused tools on virtual machines
+  package:
+    name: '{{ item }}'
+    state: absent
+  with_items: '{{ hw_vm_tools_vm_packages_remove }}'
```
Comment (karras) [inline hw_vm_tools/tasks/installation.yml]: `with_items` is no longer required, right?

File: hw_vm_tools/tasks/installation.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+
+- name: remove unused tools
+  package:
+    name: '{{ item }}'
+    state: absent
+  with_items: '{{ hw_vm_tools_packages_remove }}'
```
Comment (karras) [inline hw_vm_tools/tasks/installation.yml]: `with_items` is no longer required, right?

File: hw_vm_tools/tasks/installation.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    state: absent
+  with_items: '{{ hw_vm_tools_vm_packages_remove }}'
+  when: hw_vm_tools_vm_packages_remove is defined and
+        ansible_virtualization_role == 'guest'
+
+- name: install hardware management tools
+  package:
+    name: '{{ item }}'
+    state: present
+  with_items: '{{ hw_vm_tools_hw_packages }}'
```
Comment (karras) [inline hw_vm_tools/tasks/installation.yml]: `with_items` is no longer required, right?

File: hw_vm_tools/tasks/installation.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  when: ansible_virtualization_role == 'host'
+
+- name: install rhv management tools
+  package:
+    name: '{{ item }}'
+    state: present
+  with_items: '{{ hw_vm_tools_rhv_packages }}'
```
Comment (karras) [inline hw_vm_tools/tasks/installation.yml]: `with_items` is no longer required, right?

File: hw_vm_tools/tasks/installation.yml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  when: ansible_virtualization_role == 'guest' and
+        ansible_virtualization_type == 'RHEV'
+
+- name: install kvm management tools
+  package:
+    name: '{{ item }}'
+    state: present
+  with_items: '{{ hw_vm_tools_kvm_packages }}'
```
Comment (karras) [inline hw_vm_tools/tasks/installation.yml]: `with_items` is no longer required, right?

File: hw_vm_tools/tasks/installation.yml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  when: ansible_virtualization_role == 'guest' and
+        ansible_virtualization_type == 'kvm'
+
+- name: install vmware management tools
+  package:
+    name: '{{ item }}'
+    state: present
+  with_items: '{{ hw_vm_tools_vmware_packages }}'
```
Comment (karras) [inline hw_vm_tools/tasks/installation.yml]: `with_items` is no longer required, right?

**Review by karras — CHANGES_REQUESTED** [review]

**Comment by keachi** [comment]
You are absolutely right.

**Review by karras — APPROVED** [review]

**Pull request merged**